typeid/typeid/spec/README.md

```diff
@@ -19,7 +19,7 @@ This document formalizes the specification for TypeIDs.
 A typeid consists of three parts:
 
 1. A **type prefix**: a string denoting the type of the ID. The prefix should be
-   at most 63 characters in all lowercase snake_case ASCII `[a-z_]`.
+   at most 63 characters in all lowercase snake*case ASCII `[a-z*]`.
```

Comment:
why change this? I thought it'd be `[a-z_]`?

Reply:
Ugh, it was my editor mis-interpreting Markdown. This change shouldn't be here.